statsmodels/tsa/tests/test_stattools.py

```diff
@@ -254,13 +254,81 @@ class CheckCoint(object):
     def test_tstat(self):
         assert_almost_equal(self.coint_t,self.teststat, DECIMAL_4)
 
+# this doesn't produce the old results anymore
 class TestCoint_t(CheckCoint):
     """
     Get AR(1) parameter on residuals
     """
     def __init__(self):
-        self.coint_t = coint(self.y1, self.y2, regression ="c")[0]
+        #self.coint_t = coint(self.y1, self.y2, trend="c")[0]
+        self.coint_t = coint(self.y1, self.y2, trend="c", maxlag=0, autolag=None)[0]
         self.teststat = -1.8208817
+        self.teststat = -1.830170986148
+
+
+def test_coint():
+    nobs = 200
+    scale_e = 1
+    const = [1, 0, 0.5, 0]
+    np.random.seed(123)
+    unit = np.random.randn(nobs).cumsum()
+    y = scale_e * np.random.randn(nobs, 4)
+    y[:, :2] += unit[:, None]
+    y += const
+    y = np.round(y, 4)
+
+    for trend in []:#['c', 'ct', 'ctt', 'nc']:
+        print('\n', trend)
+        print(coint(y[:, 0], y[:, 1], trend=trend, maxlag=4, autolag=None))
```

Comment:
leave it, this is convenient for checking. I just used it to get regressiontest numbers for 'nc'